__init__.py

```diff
@@ -50,6 +51,11 @@ def get_access_token(self):
 
 
 class SpotifyConnect(spotipy.Spotify):
+    def __init__(self, skill):
+        creds = MycroftSpotifyCredentials(1)
+        super(SpotifyConnect, self).__init__(client_credentials_manager=creds)
+        self.skill = skill
+
```

Comment:
I prefer to keep the `__init__` with the vanilla Spotipy class so I can easily transfer back to it. (when and if they do a new release including the Connect API)